lib/puppet/provider/grafana_datasource/grafana.rb

```diff
@@ -147,7 +147,7 @@ def password=(value)
     save_datasource
   end
 
-  # rubocop:disable Naming/PredicateName
+  # rubocop:disable Style/PredicateName
```

Comment:
For these rubocop complains although autofix changed them?

Reply:
what did it complain about? In an older rubocop version it was in the Naming namespace and the cop got moved to Style (or the other way around).

Reply:
https://github.com/voxpupuli/puppet-grafana/runs/4368546552?check_suite_focus=true

It's not erroring though